docker/Dockerfile-flower

```diff
@@ -0,0 +1,12 @@
+FROM registry.fedoraproject.org/fedora:33
+LABEL maintainer="Red Hat"
+
+WORKDIR /src
```

Comment:
Nitpick: we don't need this line

Reply:
I am so glad you proposed better way. Thank you for everything!